src/core/manager.h

```diff
@@ -49,6 +49,7 @@ typedef enum ManagerObjective {
         MANAGER_HALT,
         MANAGER_KEXEC,
         MANAGER_SWITCH_ROOT,
+        MANAGER_SHUTDOWN,
```

Comment:
"Shutdown" is either "halt" or "poweroff". I don't think there's any reason to add another enum here.